arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'How to Deploy a Token Bridge Using the Orbit SDK'
+sidebar_label: 'Token bridge deployment'
+description: 'How to deploy a token bridge using the Orbit SDK '
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 5
+---
+
```

Comment:
It'd be good to also add here the initial note linking to the `example` in the Orbit SDK repo.

Reply:
> It'd be good to also add here the initial note linking to the `example` in the Orbit SDK repo.

I don't understand, could you please elaborate?

Reply:
Yeap, I mean the first `info` box in this page: https://arbitrum-docs-git-orbit-sdk-offchain-labs.vercel.app/launch-orbit-chain/how-tos/orbit-sdk-deploying-rollup-chain , which points to a specific example in the orbit-sdk repo.
This one is for the token-bridge: https://github.com/OffchainLabs/arbitrum-orbit-sdk/tree/main/examples/create-token-bridge-eth

Reply:
Great, thanks!